dffml/cli/ml.py

```diff
@@ -64,6 +66,13 @@ async def run(self):
         # at this point rather than an instance.
         if inspect.isclass(self.scorer):
             self.scorer = self.scorer.withconfig(self.extra_config)
+        if hasattr(self.model.config, "predict"):
+            return await accuracy(
+                self.model,
+                self.scorer,
+                self.model.config.predict,
+                self.sources,
+            )
         return await accuracy(self.model, self.scorer, self.sources)
```

Comment:
We need to modify this command to take the features to assess accuracy from the command line (via `AccuracyCMDConfig`)

Reply:
Once we do that we should only have one possible call to `accuracy()` here.

```python
        return await accuracy(self.model, self.scorer, self.features, self.sources)
```

Reply:
I suppose we dont want the cli to take predict twice, I think we should use the '-model-predict' arg for features but I'm not sure where to look in the arg parsers for that, can you point me?

Reply:
Hmmm, we really should do it twice, we had a lot of issues when we inferred the directory to save things in. We need to make sure that everything is explicate rather than implicit